apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -57,7 +68,13 @@
           </ng-icon>
         </button>
       </div>
+
+      <gn-ui-ign-api-dl
```

Comment:
The name of this component `gn-ui-ign-api-dl` is really specific to IGN. Would there be a way to make the component more generic (maybe reflecting the format it is using?)?
Maybe @jahow has a good idea about it.